Server isn't compatible with Node v0.12

hey guys, first time trying to run the boilerplate.. im on osx yosemite. 
i got thru the npm install.. but here is the console output after i run "npm start":

```
> react-boilerplate@3.0.0 start /Shared/www/tacotown/tacofinder/react-site
> npm run start:tunnel


> react-boilerplate@3.0.0 start:tunnel /Shared/www/tacotown/tacofinder/react-site
> cross-env NODE_ENV=development node server

/Shared/www/tacotown/tacofinder/react-site/server/index.js:25
app.listen(port, (err) => {
                       ^^
SyntaxError: Unexpected token =>
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3

npm ERR! Darwin 14.5.0
npm ERR! argv "node" "/usr/local/bin/npm" "run" "start:tunnel"
npm ERR! node v0.12.4
npm ERR! npm  v3.9.2
npm ERR! code ELIFECYCLE
npm ERR! react-boilerplate@3.0.0 start:tunnel: `cross-env NODE_ENV=development node server`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-boilerplate@3.0.0 start:tunnel script 'cross-env NODE_ENV=development node server'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-boilerplate package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cross-env NODE_ENV=development node server
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-boilerplate
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-boilerplate
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Shared/www/tacotown/tacofinder/react-site/npm-debug.log

npm ERR! Darwin 14.5.0
npm ERR! argv "node" "/usr/local/bin/npm" "start"
npm ERR! node v0.12.4
npm ERR! npm  v3.9.2
npm ERR! code ELIFECYCLE
npm ERR! react-boilerplate@3.0.0 start: `npm run start:tunnel`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-boilerplate@3.0.0 start script 'npm run start:tunnel'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-boilerplate package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run start:tunnel
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-boilerplate
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-boilerplate
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Shared/www/tacotown/tacofinder/react-site/npm-debug.log
```

--- a/docs/general/faq.md
+++ b/docs/general/faq.md
@@ -135,6 +135,16 @@ export default [
 ];
 ```
 
+## I'm using Node v0.12 and the server doesn't work?
+
+We settled on supporting the last three major Node.js versions for the boilerplate – at the moment
+of this writing those are v4, v5 and v6. We **highly recommend upgrading to a newer Node.js version**!
+
+If you _have_ to use Node.js 0.12, you can hack around the server not running by using `babel-cli` to
+run the server: `npm install babel-cli`, and then replace all instances of `node server` in the `"scripts"`
+in the `package.json` with `babel server`!
+
+
 ## Have another question?
 
 Submit an [issue](https://github.com/mxstbr/react-boilerplate/issues),

